sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -1083,7 +1084,7 @@ def renew_lock(self):
 
 class AMQPMessage(object):
```

Comment:
I noticed that we're laying out `param` in the docstring
```
    :param properties: Properties to add to the message.
    :type properties: ~uamqp.message.MessageProperties
    :param application_properties: Service specific application properties.
    :type application_properties: dict
    :param annotations: Service specific message annotations. Keys in the dictionary
     must be `uamqp.types.AMQPSymbol` or `uamqp.types.AMQPuLong`.
    :type annotations: dict
    :param delivery_annotations: Delivery-specific non-standard properties at the head of the message.
     Delivery annotations convey information from the sending peer to the receiving peer.
     Keys in the dictionary must be `uamqp.types.AMQPSymbol` or `uamqp.types.AMQPuLong`.
    :type delivery_annotations: dict
    :param header: The message header.
    :type header: ~uamqp.message.MessageHeader
    :param footer: The message footer.
    :type footer: dict
```

But our constructor doesn't take these.
Should them be ivar? ( or .. I don't know, maybe it's not needed as they're already annotated with `properties`)

Reply:
That was my thought.  since they have properties they're already showing up in the docs (I checked this a while back for... something, perhaps this? to see if it was needed or duplication, and my recollection is that we were in the clear.)

Reply:
Aha; I was being dense, thanks for clarifying OOB.  Yes, no params here, I had brought this over without even thinking and then took it for granted that it was appropriate.  Per our discussion, bringing these docs onto properties themselves is going to be our first iteration, and if that doesn't provide enough guidance, we can always throw up ivars or something as well, or "some enumeration at the top level" basically.